docs/EVG.md

```diff
@@ -200,6 +204,8 @@ Capitalized Terms are defined in the Baseline Requirements except where provided
 
 **Legal Entity**: A Private Organization, Government Entity, Business Entity, or Non-Commercial Entity.
 
+**Legal Entity Identifier**: The Legal Entity Identifier (LEI) is a 20-character, alpha-numeric code based on the ISO 17442 standard developed by the International Organization for Standardization (ISO). It connects to key reference information that enables clear and unique identification of legal entities participating in financial transactions. Each LEI contains information about an entity’s ownership structure and thus answers the questions of 'who is who’ and ‘who owns whom’.
```

Comment:
Similarly, marketing copy is included here.

This part is wholly unnecessary, because the only salient piece of definitional requirement, namely that it complies with ISO17442, is duplicated in the actual normative requirement.

Suggestions:
1) Delete this text entirely
2) Meaningfully define it. What are the salient bits? Possible answers include:
  * Complies with the format set forth in ISO 17442
  * Assigned by an entity approved entity
    - For example, if the intent is that it complies with the LEI Regulatory Oversight Committee's requirements and uses an LOU recognized by the LEI ROC, then let's try to find out how to say that.
    - Alternatively, it might mean assigned by an LOU accredited by GLEIF